### ISSUE TITLE
Using templates in Azure Logic Apps

### ISSUE THREAD

**Issue by freemstr** [sendgrid/sendgrid-csharp#1088]
Are there any plans to add an option to use templates with Azure Logic Apps?
.NET API takes template ID and object for paramters
Same would be great from Azure Logic Apps
Thank you.

**Comment by eshanholtz** [comment]
No, we do not currently have any plans to add templates as an option. Can you please expand on what we'd need to do to support this use case?

Thanks

**Comment by freemstr** [comment]
@eshanholtz

in .NET projects
we are able to supply template ID and data (which is effectively JSON)
which results in formatted email being sent

For example:
```
            msg.SetTemplateId(templateguid);
            msg.SetTemplateData(new { 
                body = mailbody, 
                subject = subject, 
                preheader = preheader 
            });

            var code = client.SendEmailAsync(msg)
```

Same would be great to have in Azure
After all, templating is what differentiates SendGrid from using SMTP 

From the UX perspective it would likely be an additional optional parameter on the connector for the Template Id

Thank you for looking into it

**Comment by freemstr** [comment]
If anyone is looking to do this in logic apps
![image](https://user-images.githubusercontent.com/23504818/108963238-d8622b80-7647-11eb-95fd-2a1d795a6185.png)
but building the body json could be easier done with a dedicated action 

I was not able to find other SendGrid logic apps resources on GitHub so I am posting it here as an idea and a workaround solution.
